src/integrations/contact-input.tsx

```diff
@@ -310,17 +313,23 @@ const ContactInput: FC<ContactInput> = ({
 						if (localResults.length > 0) {
 							setOptions(localResults);
 						}
-						soapFetch('AutoComplete', {
-							_jsns: 'urn:zimbraMail',
-							includeGal: 1,
-							name: e.textContent
+						soapFetch('FullAutocomplete', {
```

Comment:
I would build two different bodies depending on the presence of extraAccountsIds, calling autoComplete when is not available and the fullAutoComplete when it is avaialable

Reply:
FullAutocomplete is actually a superset of Autocomplete, if you pass no extraAccountsIds prop to the contactinput the call will be exactly as an autocomplete